pulsar-client-cpp/python/pulsar/schema/definition.py

```diff
@@ -60,6 +60,9 @@ class Record(with_metaclass(RecordMeta, object)):
     # This field is used to set namespace for Avro Record schema.
     _avro_namespace = None
 
+    # Generate a schema where fields are sorted alphabetically
+    _sorted_fields = False
```

Comment:
Do we need to set this value to `True` to avoid compatibility problems?

Reply:
My only concern is that we'll keep getting users stuck with this issue if we don't fix the default behavior.

Reply:
Maybe other users will meet new issues?